tests/test_codelists.py

```diff
@@ -260,3 +260,87 @@ def test_codelists_check_with_upstream_changes_in_CI(
     os.chdir(codelists_path)
     # check doesn't fail in CI if there are upstream errors only
     assert codelists.check()
+
+
+def test_codelists_add(codelists_path, requests_mock):
+    codelists_path /= "codelists"
+    codelists_file = codelists_path / "codelists.txt"
+    prior_codelists = codelists_file.read_text()
+    for codelist in prior_codelists.split("\n"):
```

Comment:
Using `splitlines()` rather than `split("\n")` will remove the need for the `if codelist` check on the following line.

Reply:
I copied this from one of the other tests, but I shall remember this useful distinction in future!